README.md

```diff
@@ -29,6 +29,7 @@ This library implements SPDX parsers, convertors, validators and handlers in Pyt
 - Home: https://github.com/spdx/tools-python
 - Issues: https://github.com/spdx/tools-python/issues
 - PyPI: https://pypi.python.org/pypi/spdx-tools
+- Browse the API: http://spdx.github.io/tools-python/spdx_tools.html
```

Comment:
I think I would prefer this shorter and https link: https://spdx.github.io/tools-python

Reply:
done